crypto/thread/build.info

```diff
@@ -3,19 +3,9 @@ LIBS=../../libcrypto
 $THREADS_ARCH=\
       arch.c  \
       arch/thread_win.c arch/thread_posix.c arch/thread_none.c
+$THREADS =\
+      api.c internal.c $THREADS_ARCH
 
-IF[{- !$disabled{'thread-pool'} -}]
```

Comment:
I think this test should stay. We need to get rid off the inner conditions (`IF[{- !$disabled{quic} -}]`) And make those branches un-conditional.

But I might be wrong here.

Reply:
At least some of these calls are used by rcu, which is built independently of thread-pool features.  It  think we need to include it universally

Reply:
The proposed change builds with `no-threads` as well as when `no-threads` option is committed at `Configure` command.

Reply:
agreed, I've pushed  a variant of that patch that seems to work regardless of the thread-pool and quic settings.